projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -101,14 +104,15 @@
 	display: none;
 }
 
-.contact-form label.checkbox {
+.contact-form :where(label.checkbox-multiple),
+.contact-form :where(label.radio),
+.contact-form :where(label.checkbox) {
 	font-weight: 400;
 }
```

Comment:
I think I might have found one case where this is still overriding what I set in global styles on the Option block. In my test site, I've set the Option block to have semi-bold styles, and it's working in the site editor, however on the site frontend this rule wins out. Is that because we've moved `.wp-block-jetpack-option` up a level to the wrapper, so inheriting from that font-weight loses when compared to this rule?

<img width="1041" alt="image" src="https://github.com/user-attachments/assets/4563e362-d798-41c0-b05a-7af77ee91185" />

I'm not sure what the right fix would be, but I see you've used `inherit` for the font size. Could a similar approach work for font-weight, with the default set on the wrapper? Just a thought!

Reply:
Nice catch! That should be fixed now. I did it the way you suggested by inheriting the font weight. 👍